server/index.js

```diff
@@ -68,6 +69,9 @@ app.init = function(options) {
     .use(auth.initialize)
     .use(passport.initialize())
     .use(passport.session())
+    .use(new csrf({
+      excludedMethods: ["GET", "HEAD", "OPTIONS"]
```

Comment:
this may require additional changes to secure GET requests as well, possibly add some changes to the approach itself. Going to speak with Travis